Paintroid/src/main/java/org/catrobat/paintroid/ui/DrawingSurface.kt

```diff
@@ -188,8 +188,8 @@ open class DrawingSurface : SurfaceView, SurfaceHolder.Callback {
 
     override fun surfaceChanged(holder: SurfaceHolder, format: Int, width: Int, height: Int) {
         surfaceReady = true
-        val currentToolType = toolReference.get().toolType
-        if (currentToolType != ToolType.IMPORTPNG && currentToolType != ToolType.TRANSFORM && currentToolType != ToolType.TEXT) {
+        val currentToolType = toolReference.get()?.toolType
+        if (currentToolType !== ToolType.IMPORTPNG && currentToolType !== ToolType.TRANSFORM && currentToolType !== ToolType.TEXT) {
```

Comment:
Why the "!=="? 🤔

Reply:
I thought `!==` would be required to check that equality for nullable type, but `!=` also does that job, will remove this.